apps/rehabstod/src/store/slices/settings.slice.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { UserPreferences } from '../../schemas'
```

Comment:
The dialog can't be closed with esc anymore

Reply:
Adding focus to first available input when shown seems to have solved the issue.